.github/workflows/e2e-test.yml

```
@@ -44,7 +44,7 @@ jobs:
       - name: Setup e2e test
         run: |
           cd apps/e2e
-          npx corepack enable
+          npm install -g pnpm
```

```suggestion
npm install -g corepack@latest
npx corepack enable
```
Does this code is not work?